.gitignore

```diff
@@ -40,16 +40,16 @@ bin
 *.userprefs
 
 # Build results
-[Dd]ebug/
-[Dd]ebugPublic/
-[Rr]elease/
-[Rr]eleases/
-x64/
-x86/
-bld/
-[Bb]in/
-[Oo]bj/
-[Ll]og/
+/[Dd]ebug/
```

Comment:
Is there a reason to keep all of these at the root of the project?

Reply:
I'm not sure these get created still. They are remnants from KPC.

Reply:
I should say, they don't get created on my Linux build, I'm unsure what other environments may do, so am being cautious.